python/pipeline/stack.py

```diff
@@ -977,6 +978,129 @@ def make(self, key):
         self.insert1({**key, 'resized': resized, 'lcned': lcned, 'sharpened': sharpened})
 
 
+@schema
+class StackSurfaceMethod(dj.Lookup):
+    definition = """ # Methods used to compute surface of the brain
+
+    surface_method_id   : tinyint unsigned   # Unique ID given to each surface calculation method
+    ---
+    method_title        : varchar(32)        # Title of surface calculation method
+    method_description  : varchar(256)       # Details on surface calculation
+    """
+
+    contents = [
+        [1, 'Paraboloid Fit', 'Fit ax^2 + by^2 + cx + dy + f to surface after finding max of sobel']
+    ]
+
+
+@schema
+class StackSurface(dj.Computed):
```

Comment:
i would just call it Surface: `stack.Surface` rather than `stack.StackSurface`. Same for StackSurfaceMethod